README.md

```diff
@@ -1,22 +1,25 @@
 # NYPL Global Footer
 
-This repository is the codebase to set new Footer on NYPL's landing pages.
+Static HTML and CSS files for adding the New York Public Library's footer to your app.
 
 ### Version
+1.0.0
 
+### Usage
+Copy the static HTML in `footer.html` and the CSS in `build/footer.css` and add it to your project.
+
+### Build
+The SCSS is copying over from [dgx-react-footer](https://bitbucket.org/NYPL/dgx-react-footer/src/master/). Feel free to use that if you'd like to get the source styles and related methods.
+
+You can update the SCSS and compile new CSS by running `make` at the top level of this repository. This assumes you already have sass installed on your machine.
```

Comment:
probably offering a command line there to show how to run `make`

Reply:
The command is just `make`. Seems like a bit too much for a single word command.

Reply:
Just `make`? lol OK then